### PR TITLE
add files directories to project structure

### PR DESCRIPTION
**Pull request by kien-nguyen-ngoc**
- Add _requirements.txt_, _LICENSE_, _README.md_ and _visualization_ directory to project structure when init project by command line

### CODE REVIEW

File: h1st/cli/project.py [hunk 1]
```diff
@@ -94,9 +94,11 @@ def new_project(project_name, base_path):
         data_folder = tmppath / 'data'
         model_folder = tmppath / 'models'
         notebook_folder = tmppath / 'notebooks'
+        visualization_folder = tmppath / 'visualization'
```
[the remark this answers is not in the document]
Comment (kien-nguyen-ngoc) [reply h1st/cli/project.py:97]: @cnt but if i have multi notebooks i don't want to write same thing in each `notebooks`  so create `visualization` directory contains all scripts may helps very much

[the remark this answers is not in the document]
Comment (ctn) [reply h1st/cli/project.py:97]: @kien-nguyen-ngoc I understand. How would you suggest we address this while keeping directories to a minimum? Perhaps combine `notebooks` and `visualization` into a common directory, and use a different name, like `working`, perhaps?  What directory name would you suggest?

The other suggestions are fine (LICENSE, README.md).

[the remark this answers is not in the document]
Comment (ctn) [reply h1st/cli/project.py:97]: @kien-nguyen-ngoc do you want to update the edits, or shall I? There's another directory that we are logically adding for audit-trailing.

**Comment by ctn** [comment]
@nqbao please finalize the changes as you see fit and I'll review & merge. We need this PR closed.

**Comment by nqbao** [comment]
Since #40 is merged, let me work on this

**Comment by nqbao** [comment]
Hi @kien-nguyen-ngoc , would you like to continue to work on this?

**Comment by nqbao** [comment]
I'm going to fork this in my repo to continue to work on this.

**Comment by ctn** [comment]
@nqbao please use your judgement to incorporate the most appropriate suggestions, and then close the PR.

**Comment by Shiti** [comment]
The `cli` module has been removed.

**Pull request closed without merging**